File: 1/1.java
import java.util.Arrays;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length;
        for (int a = 0; a < n-1; a++){
            for (int b = a+1; b < n; b++){
                if ((nums[a]+nums[b]==target)){
                    return new int[]{a, b};
                }
            }
        }
        return new int[]{0,0};
    }
}

File: 1/1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for n in range(len(nums)-1):
            if (target-nums[n]) in nums[n+1:]:
                return [n, n+1 + nums[n+1:].index((target-nums[n]))]
